common/helpers_test.go

```diff
@@ -9,18 +9,17 @@ import (
 
 	"github.com/chromedp/cdproto/cdp"
 	"github.com/chromedp/cdproto/runtime"
-	"github.com/dop251/goja"
 	"github.com/stretchr/testify/require"
 
 	"github.com/grafana/xk6-browser/log"
 )
 
-func newExecCtx() (*ExecutionContext, context.Context, *goja.Runtime) {
+func newExecCtx() (*ExecutionContext, context.Context) {
 	ctx := context.Background()
 	logger := log.NewNullLogger()
 	execCtx := NewExecutionContext(ctx, nil, nil, runtime.ExecutionContextID(123456789), logger)
 
-	return execCtx, ctx, goja.New()
```

Comment:
Removed `goja.New` here because `ExecutionContext` no longer depends on Goja 🎉

Reply:
This is big 🙌